### ISSUE TITLE
I Got banned ☝️🤓

### ISSUE THREAD

**Issue by Zacki213** [poketwo/poketwo#576]
Hi, how are you? So I was fine logging back into Discord when I came across this image 

![Screenshot_20231230-222808~2](https://github.com/poketwo/poketwo/assets/155275188/bcc3fbae-18aa-469e-a5f1-ce5b3c0331ac)

I was surprised and a little sad at the same time, because just so you know I never messed around with this kind of thing (hacks, bugs, scripts, etc.) Mainly on Discord, but for you to have a context I I was talking to my friend about auto catch. (I'm Brazilian so the prints are not translated XD)

![Screenshot_20231230-223906~2](https://github.com/poketwo/poketwo/assets/155275188/4ebacafa-f131-47db-ae31-f6834950f158)
![Screenshot_20231230-223738~2](https://github.com/poketwo/poketwo/assets/155275188/3c2b29d1-112f-4503-92cc-687c19a04b97)
![Screenshot_20231230-223859~2](https://github.com/poketwo/poketwo/assets/155275188/3281c70c-73f7-4f18-9d13-6df65e4668ab)

To sum up the images: I was too lazy to catch Pokémon so I researched auto catch, I ended up finding one but it was paid, so I left it alone. But apparently someone saw this And report me.

I would have left it alone, but I at least want to know who reported me, or what actually caused my ban? Because from what I saw in the terms and services, you are only banned if you use it, not if you try to use it. For me, it doesn't make any sense to do that. But if I'm wrong just tell me.

In short, I would just like to know why and if there is a way to reverse this ban, I am waiting for answers. Thank you, have a good end of the year!

**Comment by WitherredAway** [comment]
Generally showing interest in using an autocatcher is enough to get you suspended. Please appeal for your suspension at https://forms.poketwo.net/a/suspension-appeal and explain the situation in detail along with evidence that you did in fact not use an autocatcher following that conversation.
